rxlifecycle/src/main/java/com/trello/rxlifecycle/ActivityLifecycleProvider.java

```diff
@@ -8,7 +8,7 @@
  * Common interface for all RxActivity extensions.
  *
  * Useful if you are writing utilities on top of rxlifecycle-components,
- * or you are implementing your own component not supported in this library.
+ * or if you are implementing your own component which is not supported in this library.
```

Comment:
AFAIK this sentence was valid English in the first place. But if you really want to fix it up, you'd have it split the "or" on the verb:

> Useful if you are writing utilities on top of rxlifecycle-components or implementing your own component not supported by this library.